Source/ablastr/utils/Enums.H

```diff
@@ -0,0 +1,38 @@
+/* Copyright 2024 Axel Huebl
+ *
+ * This file is part of ABLASTR.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+
+#ifndef ABLASTR_UTILS_ENUMS_H_
+#define ABLASTR_UTILS_ENUMS_H_
+
+#include <AMReX_BaseFwd.H>
+#include <ablastr/utils/Enums.H>
+
+#include <string>
+
+namespace ablastr::utils::enums
+{
+    /** Type of grids used in a simulation:
+     *
+     * Collocated at the same location (AMReX: all "NODAL"), staggered (Yee-style), or hybrid.
+     */
+    enum struct GridType {
+        Collocated = 0,
+        Staggered = 1,
+        Hybrid = 2
+    };
+
+    /** Mesh-refinement patch
+     *
+     * The current (fine) or lower-level (coarse) resolution.
+     */
```

Comment:
@EZoni @RemiLehe @oshapoval pls let me know if these doc strings are reasonable or if you would change something :pray: